robottelo/manifests.py

```diff
@@ -17,160 +12,103 @@
 from robottelo.config import settings
 
 
-def get_tempfile():
-    """Creates a temporary file.
-
-    Generates temporary file with mkstemp,
-    closes the handle, and returns the path to temporary file.
-
+class ManifestCloner(object):
+    """Manifest clonning utility class."""
+    def __init__(self, template=None, signing_key=None):
+        self.template = template
+        self.signing_key = signing_key
+
+    def _download_manifest_info(self):
+        """Download and cache the manifest information."""
+        self.template = requests.get(settings.fake_manifest.url).content
+        self.signing_key = requests.get(settings.fake_manifest.key_url).content
+        self.signature = PKCS1_v1_5.new(RSA.importKey(self.signing_key))
```

Comment:
We will not support DSA unless the way manifest content are signed uses it. This signature is being used to just sign the contents of the manifest.